src/app/products/components/product-delete-confirm-dialog/product-delete-confirm-dialog.component.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { HarnessLoader } from "@angular/cdk/testing";
+import { TestbedHarnessEnvironment } from "@angular/cdk/testing/testbed";
+import { Component } from "@angular/core";
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { MatButtonModule } from "@angular/material/button";
+import { MatDialog, MatDialogModule } from "@angular/material/dialog";
+import { MatDialogHarness } from "@angular/material/dialog/testing";
+import { NoopAnimationsModule } from "@angular/platform-browser/animations";
+import { take } from "rxjs/operators";
+
+import { Product } from "src/app/shared/models/product";
+import { ProductDeleteConfirmDialogComponent } from "./product-delete-confirm-dialog.component";
+import { ProductDeleteConfirmDialogHarness } from "./product-delete-confirm-dialog.harness";
+
+const product = {
+  name: "test product",
+  description: "test description",
+  picture: "picture url"
+}
+
+describe("AppModule => ProductDeleteConfirmDialogComponent", () => {
```

Comment:
Leave just the component name.

Reply:
Mentioned change is completed hence resolving this conversation.